src/org/ods/services/BitbucketService.groovy

```diff
@@ -153,7 +240,7 @@ class BitbucketService {
                 label: "Post comment to PR#${pullRequestId}",
                 script: """curl \\
                   --fail \\
-                  --silent \\
+                  -sS \\
```

Comment:
@michaelsauter looking at some of the PRs on master which are not in 3.x to understand what we'll merge into soon. Wasn't -sS source of some issues recently?

Reply:
No this is an improvement - the issues were that `-S` was missing :)

`-s` is the same as `--silent`, and `-S/--show-error` enables error output, see `man curl`:

> When used with -s, --silent, it makes curl show an error message if it fails